packages/apollo-server-core/src/requestPipelineAPI.ts

```diff
@@ -1,3 +1,16 @@
+import {
```

Comment:
I thought the whole point of this file was it couldn't import things... Ah, neat, you fixed that a while ago in #2990. That said, that PR described this file as just being here for backwards compatibility. Is it actually the best place for this new code, rather than, I dunno, schemaInstrumentation.ts?